### ISSUE TITLE
Travis checking for all README files

### ISSUE THREAD

**Issue by JimmyCarlsson** [perma-id/w3id.org#320]
I noticed as I tried to create a pull-request that the travis-ci check failed to complete its validation of README links, but not due to my own commit, but to another users link timing out.

Maybe the travis check should focus on the diff of the pull request rather than checking all README files? As the service grows I can imagine that at least one link will be dead at all times.

**Comment by davidlehn** [comment]
Yes, this would be a nice change.  If anyone knows how to do it, please write a patch!
